features/step_definitions/export.ts

```diff
@@ -79,7 +80,21 @@ Then(
           return `"${field}"`;
         })
         .join(",");
-      assert.match(this.response.stdout, new RegExp(`${values}`));
+      assert.match(this.response.stdout.toString(), new RegExp(`${values}`));
+    });
+  },
+);
+
+Then(
+  "The output message with {string} encoded should match with the data below:",
+  async function (encoding, table) {
+    const decodedOutputMsg = iconv.decode(this.response.stdout, encoding);
+    const records = this.replacePlaceholdersInRawDataTables(table.raw());
+    records.forEach((record: string[]) => {
+      const values = record
+        .map((field: string) => (field ? `"${field}"` : ""))
+        .join(",");
```

Comment:
Can we make these lines reusable?
It is the same as the step "The output message should match the data in the order as below:"